### DISCUSSION TITLE
Ask for authorization to reuse partial text from you docs

### DISCUSSION THREAD

**Discussion by Tartifless** [libretro/docs#768] [General]
Hello Libretro team,

I am part of the Retrobat front-end team.
We are currently building our wiki, along with the next V5.

As part of the wiki, i created an Arcade Guide page, where i would like to reuse 2 elements from your "Getting started with Arcade page":
https://docs.libretro.com/guides/arcade-getting-started/
The elements i'd like to reuse are:
- The arcade terminology (with slight changes)
- The type of rom sets

You can see the page in question here, and the reference at the bottom to your page:
https://wiki.retrobat.org/systems-and-emulators/arcade-guide

Is this ok for you or shall i rewrite it ?

Regards,
Ben S.

**Comment by hizzlekizzle** [comment]
The docs are licensed MIT, so as long as we're credited, you should be good to go either way: https://github.com/libretro/docs/blob/master/LICENSE

**Reply by Tartifless** [reply]
As you can see, I credited and Linked to the page.
Thanks and good continuation to you !

**Reply by hizzlekizzle** [reply]
ah, yeah, looks great! cheers!
